javascript/src/main/pages/Home/Home.js

```diff
@@ -1,9 +1,8 @@
 import React from "react";
 import { useState } from "react";
 import { Jumbotron } from "react-bootstrap";
-import { useAuth0 } from "@auth0/auth0-react";
-import { Redirect } from "react-router-dom";
 import BasicCourseSearchForm from "../../components/BasicCourseSearch/BasicCourseSearchForm";
+import BasicCourseTable from "../../components/BasicCourseSearch/BasicCourseTable";
 import JSONPrettyCard from "../../components/Utilities/JSONPrettyCard";
```

Comment:
We should try to stick to absolute paths instead of relative ones.

Reply:
Fixed